lldb/test/Shell/Register/riscv64-gp-read.test

```diff
@@ -0,0 +1,38 @@
+# REQUIRES: native && target-riscv64
+# RUN: %clangxx_host -fomit-frame-pointer %p/Inputs/riscv64-gp-read.cpp -o %t
```

Comment:
I'm guessing that `-fomit-frame-pointer` is to stop code after the inline asm generating save/restore routines, but if you were to stop during the inline asm, this wouldn't be needed.

You could add something like "brk \n\t" to the end and lldb will stop there, it won't know who put the breakpoint in but that doesn't matter here. brk is the AArch64 mnemonic I think but look around you'll find the RISC-V one somewhere "software breakpoint" is the term you want to find.